Library/Homebrew/os/linux/elf.rb

```diff
@@ -148,13 +158,42 @@ def needed_libraries_using_patchelf_rb(path)
   end
   private_constant :Metadata
 
+  def save_using_patchelf(new_interpreter, new_rpath)
+    patchelf = DevelopmentTools.locate "patchelf"
+    odie "Could not locate patchelf, please: brew install patchelf." if patchelf.nil?
+    pef_args = []
+    pef_args.push("--set-interpreter", new_interpreter) if new_interpreter
+    pef_args.push("--force-rpath", "--set-rpath", new_rpath) if new_rpath
+
+    Homebrew.safe_system(patchelf, *pef_args, to_s)
```

Comment:
```suggestion
    odie "Could not locate patchelf, please: brew install patchelf." if patchelf.blank?
    
    args = []
    args << "--set-interpreter" << new_interpreter if new_interpreter.present?
    args << "--force-rpath" << "--set-rpath" << new_rpath if new_rpath.present?

    Homebrew.safe_system(patchelf, *args, to_s)
```

Reply:
@MikeMcQuaid what's the advantage of using `<<` over `.push`

Reply:
No real advantage, just we use it more commonly

Reply:
makes sense, thanks.